### PR TITLE
Update mkdocs material and centralize version number

### PR DESCRIPTION
**Pull request by RDIL**

### CODE REVIEW

**Comment by fkorotkov** [comment]
You need to update `MKDOCS_VERSION` in `.cirrus.yml`. Ideally we should parse the version from there.

**Comment by RDIL** [comment]
I did this so you can also still run the scripts in the devenv

File: scripts/build.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-docker run --rm -it -v ${PWD}:/docs squidfunk/mkdocs-material:4.4.2 build
+docker run --rm -it -v ${PWD}:/docs squidfunk/mkdocs-material:{cat theme/.material-version.cfg} build
```
Comment (fkorotkov) [inline scripts/build.sh]:
```suggestion
docker run --rm -it -v ${PWD}:/docs squidfunk/mkdocs-material:$(grep 'MKDOCS_VERSION:' .cirrus.yml | cut -d\  -f4) build
```

File: scripts/start.sh [hunk 2]
```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material:4.4.2
+docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material:{cat theme/.material-version.cfg}
```
Comment (fkorotkov) [inline scripts/start.sh]:
```suggestion
docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material:$(grep 'MKDOCS_VERSION:' .cirrus.yml | cut -d\  -f4)
```

**Comment by RDIL** [comment]
Bump for rereview/merge

**Pull request merged**